src/ExpandObjects/epfilter.f90

```diff
@@ -14227,6 +14227,7 @@ SUBROUTINE CreateNewUnitary
     CALL AddToObjFld('Rated COP', base + usCoolCoilCOPOff,' ')
     CALL AddToObjStr('Rated Air Flow Rate {m3/s}', 'autosize')
     CALL AddToObjStr('Rated Evaporator Fan Power per Volume Flow Rate {W/(m3/s)}','')
+    CALL AddToObjStr('2023 Rated Evaporator Fan Power Per Volume Flow Rate {W/(m3/s)}','')
```

Comment:
For cooling and heating the existing field names should be "2017 Rated ..." if I'm following correctly.

Reply:
Yep fixed it to use the new 2017 field name.  Tried to be very consistent with the exact field names in the transition rules description file.